src/main/java/org/ohdsi/circe/cohortdefinition/negativecontrols/OutcomeCohortExpression.java

```diff
@@ -16,17 +17,16 @@
  @JsonProperty("detectOnDescendants")
  public Boolean detectOnDescendants = true;
  
- @JsonProperty("domainIds")
- public List<String> domainIds = new ArrayList<>(); 
+ @JsonProperty("domains")
+ public List<String> domains = new ArrayList<>(); 
  
- /*
-	public static CohortExpression fromJson(String json) {
-		try {
-			CohortExpression expression = JSON_MAPPER.readValue(json, CohortExpression.class);
-			return expression;
-		} catch (Exception e) {
-			throw new RuntimeException("Error parsing cohort expression", e);
-		}
-	}
- */
+ public static OutcomeCohortExpression fromJson(String json) {
+   try {
```

Comment:
https://github.com/OHDSI/StandardizedAnalysisAPI/blob/pathways-analysis/src/main/java/org/ohdsi/analysis/Utils.java#L67

Reply:
That makes sense to me @pavgra; however, I'm not sure if we're leveraging SAA in circe just yet? I'll leave it to @chrisknoll's discretion on how to proceed.

Reply:
Post Symposium.

Reply:
To make sure I'm clear: we can refactor to use the SAA Utils for formatting after symposium. Can we merge this change in with that understanding? I see @pavgra linked it to the WebAPI issue. I need this functionality for Symposium so if we can merge it in, that would be greatly appreciated. @schuemie needs this for testing on Hydra.

Reply:
Ok, it will be done.